tests/integration/main.go

```diff
@@ -29,12 +28,55 @@ func main() {
 
 	for serverName, binCommand := range expectedMCPServers {
 		expectedOutput := fmt.Sprintf("%s: npx -y %s (stdio) - Connected", serverName, binCommand)
-		if strings.Contains(string(output), expectedOutput) {
-			fmt.Printf("✅ Assertion passed: Output contains the connected %s server line.\n", serverName)
-		} else {
-			fmt.Printf("❌ Assertion failed: Output did not contain the connected %s server line.\n", serverName)
-			os.Exit(1)
+		if !strings.Contains(string(output), expectedOutput) {
+			return fmt.Errorf("assertion failed: output did not contain the connected %s server line", serverName)
 		}
+		fmt.Printf("✅ Assertion passed: Output contains the connected %s server line.\n", serverName)
+	}
+	return nil
+}
+
+func testCallGcloudMCPTool() error {
+	fmt.Println("🚀 Starting gcloud-mcp tool call integration test...")
+	args := []string{"gcloud-mcp"}
```

Comment:
When would we be invoking client.InvokeMCPTool with more arguments than the server name?

Reply:
Cool - thank you for clarifying.